src/dd-draggable.ts

```diff
@@ -193,6 +183,11 @@ export class DDDraggable extends DDBaseImplement implements HTMLElementExtendOpt
         this._callDrag(e);
       }
     } else if (Math.abs(e.x - s.x) + Math.abs(e.y - s.y) > 3) {
+      let node = (this.el as GridItemHTMLElement)?.gridstackNode;
+      if (node) {
+        const rect = this.el.getBoundingClientRect();
+        node._originalMousePositionInsideElement = { x: e.clientX - rect.left, y: e.clientY - rect.top };
```

Comment:
why do you have this code at all (shouldn't be on node, but the draggable class only) since you have same line 205. delete from GridStackNode